spec/spec_helper.rb

```diff
@@ -32,6 +32,10 @@ def image_path(path, options = {})
 # include the manageiq-gems-pending matchers
 Dir[ManageIQ::Gems::Pending.root.join("spec/support/custom_matchers/*.rb")].each { |f| require f }
 
+# To be extracted with embedded_ansible
+require ManageIQ::Providers::AnsibleTower::Engine.root.join("spec/support/vcr_helper.rb").to_s
+Dir[ManageIQ::Providers::AnsibleTower::Engine.root.join("spec/support/ansible_shared/**/*.rb")].each { |f| require f }
+
```

Comment:
I am :+1: for this as it's a very prescriptive change, and is clearly labeled to be moved when embedded_ansible is extracted.